src/redux/book/booksSlice.js

```diff
@@ -42,6 +42,18 @@ const bookSlice = createSlice({
         item_id,
         ...action.payload[item_id][0],
       }));
+
+      console.log(action.payload);
+      console.log('get data successfully');
+      console.log(action.payload);
+    });
+    builder.addCase(fetchBooks.rejected, () => {
+      console.log('can not get data');
+    });
+    builder.addCase(addBooks.fulfilled, (state, action) => {
+      state.totalbooks.push(action.payload);
+      console.log('Add a book successfully');
+      console.log(action.payload);
```

Comment:
- [ ] Good work so far working on the requested changes from the previous reviewer. However, you still have some changes to do to get the approval 👍 Please delete any `console` statement as it's not [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log) and also harms the optimization of the app and makes the code un-pretty 😉